client/service/util.go

```diff
@@ -0,0 +1,31 @@
+package service
+
+import (
+	clientError "github.com/scalar-labs/scalardl-go-client-sdk/v3/client/error"
+	"github.com/scalar-labs/scalardl-go-client-sdk/v3/ledger/statuscode"
+	"github.com/scalar-labs/scalardl-go-client-sdk/v3/rpc"
+	"google.golang.org/grpc/metadata"
+	"google.golang.org/protobuf/proto"
+)
+
+func getClientErrorFromTrailer(trailer metadata.MD) (err clientError.ClientError) {
```

Comment:
This function is an internal function used to retrieve Scalar DL Status Code from gRPC's metadata.